### PR TITLE
Lakshay Vardhan's json file was added to LinkFree

### PR DESCRIPTION
**Pull request by iwantamaste**

### CODE REVIEW

File: public/data/LakshayVardhan.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/iwantamaste",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/VardhanLakshay_",
+      "icon": "twitter"
+    },
```
Comment (CodingSpecies) [inline public/data/LakshayVardhan.json:15]: @iwantamaste please can you remove the comma after the bracket.

File: public/data/LakshayVardhan.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  ],
```
Comment (CodingSpecies) [inline public/data/LakshayVardhan.json:16]: Please remove the bracket.

File: public/data/LakshayVardhan.json (hunk 1 above)
Comment (CodingSpecies) [inline public/data/LakshayVardhan.json:15]: Please swap line 15 and 16.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Review by CodingSpecies — APPROVED** [review]
Thank you @iwantamaste

File: public/data/LakshayVardhan.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/VardhanLakshay_",
+      "icon": "twitter"
+    }
+  ]
+}
+
```
Comment (eddiejaoude) [inline public/data/LakshayVardhan.json:18]: GitHub is highlighting that this needs an extra blank line - we can merge and this can be fixed later

**Pull request merged**